### PR TITLE
Add jdk_osb target

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

File: openjdk/playlist.xml [hunk 1]
```diff
@@ -36,6 +36,28 @@
...
+		<testCaseName>jdk_osb</testCaseName>
+		<variations>
+			<variation>Mode150</variation>
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) $(CUSTOM_NATIVE_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS) $(VMOPTION_HEADLESS)$(Q) \
+	$(TIMEOUT_HANDLER) \
+	-w $(Q)$(REPORTDIR)$(D)work$(Q) \
```
Comment (sendaoYan) [inline openjdk/playlist.xml:49]: Whether to consider uniformly pointing the jtreg test report path to REPORTDIR
-r $(Q)$(REPORTDIR)$(D)report$(Q)

File: openjdk/playlist.xml [hunk 2]
```diff
@@ -36,6 +36,28 @@
...
+	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
```
Comment (sendaoYan) [inline openjdk/playlist.xml:50]: Whether to consider uniformly pointing the jtreg test report path to REPORTDIR
-r $(Q)$(REPORTDIR)$(D)report$(Q)

Comment (llxia) [reply openjdk/playlist.xml:50]: Yes, I will update it. Thanks

**Comment by llxia** [comment]
This is moved to vendor repo.

**Pull request closed without merging**